website/code_issues.qmd

```diff
@@ -319,3 +319,44 @@ Calls of `rm(list = ls())` to remove variables of the current environment, shoul
 
 :::
 
+
+
+
+---------------------------------------------
+
+
+# Calling `installed.packages()`
+
+## Problem
+
+You are calling `installed.packages()` to check if a package is installed.
+
+## Solution
+
+Instead of using `installed.packages()`, use `requireNamespace("pkg")` or `require("pkg")` to find out if packages are available.
+
+### Details
+
+::: {.callout-note title="CRAN Review Communication" appearance="simple" icon=false collapse=true}
```

Comment:
<img width="931" alt="Screenshot 2024-10-02 at 10 39 14 AM" src="https://github.com/user-attachments/assets/9f239843-3308-45df-adfe-08ad8b1c3466">

I'm also not seeing the callout format rendered correctly.

Reply:
Weird, I'll look into this and fix it

Reply:
It seems to work in the rendered webpage. No idea what went wrong in your render